2018/07.md

```diff
@@ -106,6 +106,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | 1     | 30m     | [RegExp match offsets](https://github.com/tc39/proposal-regexp-match-offsets) for Stage 2 | Ron Buckton |
     | 1     | 60m     | [Partial Application](https://github.com/tc39/proposal-partial-application) Update | Ron Buckton |
     | 1     | 60m     | [Cancellation](https://github.com/tc39/proposal-cancellation) Update | Ron Buckton |
+    | 0     | 60m     | [Dynamic Modules Proposal](https://github.com/guybedford/proposal-dynamic-modules) for Stage 1 | TBD |
```

Comment:
i think for this meeting we were trying for stage 0 proposals to have 10 minute lightning talks (see the next section). do you anticipate this taking an hour to determine "the committee expects to devote time to examining the problem space, solutions and cross-cutting concerns"?

Reply:
BTW you can make a longer than 10 minute talk for a Stage 0 proposal, it'll just be prioritized later on the agenda.